### PR TITLE
E2E: Add markers linux and windows

### PR DESCRIPTION
**Pull request by BelenValdivia**
|Related issue|
|-------------|
|     https://github.com/wazuh/wazuh-qa/issues/3204        |

### Linux
1. Run test as follows:

```
python -m pytest tests/end_to_end/test_basic_cases/ -m linux --inventory_path=/home/belen/Repositories/wazuh-qa/tests/end_to_end/general_playbooks/inventory_jenkins.yaml --html=3204-T1-R${i}-e2e-belenvaldivia.html --self-contained-html
```

### Windows
1. Run test as follows:

```
python -m pytest tests/end_to_end/test_basic_cases/ -m win32 --inventory_path=/home/belen/Repositories/wazuh-qa/tests/end_to_end/general_playbooks/inventory_jenkins.yaml --html=3204-T1-R${i}-e2e-belenvaldivia.html --self-contained-html
```
---

### Linux

| Tester             | Test path | Jenkins | Local  | OS | Commit | Notes                |
|--------------------|-----------|---------|--------|-----|--------|----------------------|
| @BelenValdivia  (Developer)  |          |  ⚫⚫⚫  | [🟢](https://github.com/wazuh/wazuh-qa/files/9455468/T1-R1-3204-e2e-linux.zip)  [🟢](https://github.com/wazuh/wazuh-qa/files/9455469/T1-R2-3204-e2e-linux.zip)  [🟢](https://github.com/wazuh/wazuh-qa/files/9455470/T1-R3-3204-e2e-linux.zip)   |         |         | Nothing to highlight |
| @user (Reviewer)   |           | ⚫⚫⚫ | :no_entry_sign: :no_entry_sign: :no_entry_sign:  |        |         | Nothing to highlight |

### Windows

| Tester             | Test path | Jenkins | Local  | OS | Commit | Notes                |
|--------------------|-----------|---------|--------|-----|--------|----------------------|
| @BelenValdivia  (Developer)  |          |  ⚫⚫⚫  | [🟢](https://github.com/wazuh/wazuh-qa/files/9455479/T1-R1-3204-e2e-windows.zip)  [🟢](https://github.com/wazuh/wazuh-qa/files/9455480/T1-R2-3204-e2e-windows.zip)  [🟢](https://github.com/wazuh/wazuh-qa/files/9455481/T1-R3-3204-e2e-windows.zip)   |         |         | Nothing to highlight |
| @user (Reviewer)   |           | ⚫⚫⚫ | :no_entry_sign: :no_entry_sign: :no_entry_sign:  |        |         | Nothing to highlight |

### CODE REVIEW

File: tests/end_to_end/test_basic_cases/test_ip_reputation/test_ip_reputation.py [hunk 1]
```diff
@@ -59,6 +60,9 @@
 
 configurations, configuration_metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
 
+# Marks
+pytestmark = [TIER0, LINUX]
```
Comment (juliamagan) [inline tests/end_to_end/test_basic_cases/test_ip_reputation/test_ip_reputation.py:64]: This test needs a Windows agent, too.

**Review by juliamagan — CHANGES_REQUESTED** [review]

File: tests/end_to_end/test_basic_cases/test_ip_reputation/test_ip_reputation.py (hunk 1 above)
Comment (BelenValdivia) [reply tests/end_to_end/test_basic_cases/test_ip_reputation/test_ip_reputation.py]: Done in d5687d0bf34e45289c3be8be4bb4e3382571a2e2

**Review by juliamagan — APPROVED** [review]

File: tests/end_to_end/test_basic_cases/test_audit/test_audit.py [hunk 2]
```diff
@@ -57,6 +58,9 @@
 
 configurations, configuration_metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
 
+# Marks
+pytestmark = [TIER0, LINUX]
```
Comment (jmv74211) [inline tests/end_to_end/test_basic_cases/test_audit/test_audit.py:62]:
```suggestion
pytestmark = [TIER_0, LINUX]
```

File: tests/end_to_end/test_basic_cases/test_audit/test_audit.py [hunk 3]
```diff
@@ -46,6 +46,7 @@
 from wazuh_testing import end_to_end as e2e
 from wazuh_testing import event_monitor as evm
 from wazuh_testing.tools import configuration as config
+from wazuh_testing.modules import TIER0, LINUX
```
Comment (jmv74211) [inline tests/end_to_end/test_basic_cases/test_audit/test_audit.py:49]:
```suggestion
from wazuh_testing.modules import TIER_0, LINUX
```

**Review by jmv74211 — CHANGES_REQUESTED** [review]

**Review by jmv74211 — APPROVED** [review]

**Pull request merged**